src/main/java/io/tus/java/client/TusClient.java

```diff
@@ -52,6 +54,24 @@ public URL getUploadCreationURL() {
         return uploadCreationURL;
     }
 
+    /**
+     * Set the upload proxy that will be used with the {@link #uploadCreationURL}.
```

Comment:
Can we just call it proxy everywhere instead of upload proxy? I was wondering what the difference between an upload proxy and proxy was.